### DISCUSSION TITLE
Using VueJS and Shopify's Theme Editor with Dynamic Sections

### DISCUSSION THREAD

**Discussion by sebszocinski** [uicrooks/shopify-theme-lab#61] [Q&A]
Hi, I've been working on a Shopify theme using VueJS and ran into an issue with having `content_for_layout` / `content_for_index` inside the Vue instance and then being able to use dynamic sections in the Theme Editor.

You seem to have got around this somehow and I can't work out how you did it? 

I'd love to switch to Theme Lab but it would be a fair amount of work so i'm trying to work out how you managed to have Vue play nicely with the Theme Editor.

Any tips or advice would be greatly appreciated.

**Comment by sebszocinski** [comment]
Never mind, I found the answer here #14 I spent 4 hours debugging this 🤦‍♂️

**Reply by sergejcodes** [reply]
Here's another example of [settings_data.json](https://github.com/uicrooks/shopify-foundation-theme/blob/master/shopify/config/settings_data.json) and [dynamic sections](https://github.com/uicrooks/shopify-foundation-theme/tree/master/shopify/sections) implemented in a Shopify Theme Lab theme.
